src/vos/vos_common.c

```diff
@@ -360,6 +361,25 @@ vos_init(void)
 	if (rc)
 		D_GOTO(exit, rc);
 
+	evt_mode = getenv("DAOS_EVTREE_MODE");
+	if (evt_mode) {
+		if (strcasecmp("soff", evt_mode) == 0)
+			vos_evt_feats = EVT_FEAT_SORT_SOFF;
+		else if (strcasecmp("dist_even", evt_mode) == 0)
+			vos_evt_feats = EVT_FEAT_SORT_DIST_EVEN;
+	}
+	switch (vos_evt_feats) {
+	case EVT_FEAT_SORT_SOFF:
+		D_INFO("Using start offset sort for evtree\n");
+		break;
+	case EVT_FEAT_SORT_DIST_EVEN:
+		D_INFO("Using distance sort sort for evtree with even split\n");
+		break;
+	default:
+		D_ASSERT(vos_evt_feats == EVT_FEAT_SORT_DIST);
```

Comment:
The D_ASSERT() is redundant since we have no "else" branch after above "else if" at line 368.

Reply:
It was more of a sanity check  but I removed it